liferay-lima/liferay-portal
# LPS-189476 Copy tag and categories when coping a File

Follow up from https://github.com/liferay-lima/liferay-portal/pull/4444
LPS-187728 is needed a rebase so > _on hold_


https://liferay.atlassian.net/browse/LPS-189476


To test it :
For all:  Enable FF: LPS-182512

1. Go to global site
2. On this create a category and a tag
3. Create a Document with created category and tag
4. Go to document > dropdown > copy to 
5. copy to Liferay site (home)
6. Go to Liferay site > Documents and Media
7. check document copied from global (edit) has the tag and the category


other test

1. Create an Asset Library
2. Connect to Liferay Site
3. On the asset library create a category and a tag
4. Create a Document with created category and tag
5. Go to document > dropdown > copy to 
6. copy to Liferay site (home)
7. Go to Liferay site > Documents and Media
8. check document copied from asset library has the tag and the category

note : if copy between sites not related, only common categories and tags (from global or an common ancestor) is copied





- LPS-189476 copy the tags and the categories of the source file entry
- LPS-189476 add the groups ids to copyFileEntry to check if the categories and tags are allowed on the groups and can be added to the new file
- LPS-189476 buildService
- LPS-189476 add the connected groups to check if the tag/category is allowed to copy
- LPS-189476 semver
